client/src/pricing_mod1/ModOne.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import PricePerNight from './PricePerNight';
+import Rating from './Rating';
+
+export const Holder = styled.div`
+  display: flex;
+  justify-content: center; 
+  align-items: center;
+  flex-direction: column;
+  width: inherit;
+  width: 250px;
+  border-bottom: 1px solid rgb(172, 172, 172);
+  font-weight: bold;
+`;
+Holder.displayName = 'Holder';
+
+export default function ModOne( props ) {
+  ModOne.propTypes = {
+    price: PropTypes.number,
+    rating: PropTypes.number,
+    numReviews: PropTypes.number,
+
+  };
+  ModOne.defaultProps = {
```

Comment:
Nice use of defaultProps in case for some reason they aren't passed down!  You taught me something here, I may incorporate into my own code.

Reply:
The linter yells at you if you don't do it lol